src/screens/IndexScreen.js

```diff
@@ -616,9 +653,10 @@ export const IndexScreen = ({ navigation, route }) => {
           }}
         </Query>
       )}
-      {query === QUERY_TYPES.POINTS_OF_INTEREST && (
-        <IndexMapSwitch filter={filterType} setFilter={setFilterType} />
-      )}
+      {query === QUERY_TYPES.POINTS_OF_INTEREST &&
+        filterType.find((i) => i.selected && i.title === 'Listenansicht') && (
```

Comment:
the logic you implemented with `find` returns an object or not. luckily that works and resolves to true. i suggest a refactoring to really rely on a boolean here: `filterType.find((entry) => entry.title == texts.locationOverview.list)?.selected`